workspaces/azure-devops/.changeset/giant-goats-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-azure-devops-backend': minor
+---
+
+Fixed bug in plugin-azure-devops-backend where proper error was not thrown when gitRepository was not found.
```

Comment:
```suggestion
Fixed a bug where proper error was not thrown when the repository was not found.
```

The changelog this will get added to will have the name of the plugin so we don't need it here and we can simplify the wording a little.